source/atlas-cli-deploy-docker.txt

```diff
@@ -0,0 +1,148 @@
+.. _atlas-cli-deploy-docker:
+
+=========================================
+Create a Local Atlas Deployment in Docker
+=========================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial shows you how to use the ``atlas deployments`` command to 
+create a local |service| deployment in Docker. In this tutorial, we 
+will deploy a single-node replica set in Docker.
+
+.. include:: /includes/fact-atlas-deployments-preview.rst
+
+.. _atlas-cli-deploy-local-reqs:
+
+Complete the Prerequisites
+--------------------------
+
+.. include:: /includes/fact-deploy-tutorial-prereqs.rst
+
+.. _atlas-cli-deploy-docker-setup:
+
+Create a Local Atlas Deployment in Docker
+-----------------------------------------
+
+Use the ``atlas deployments`` command to create a local |service|
+deployment in Docker.
+      
+.. procedure::
+   :style: normal
+
+   .. step:: Install and start Docker.
+
+      To learn more, see the `Docker documentation <https://docs.docker.com/desktop/install/mac-install/>`__.
+
+   .. step:: Pull down the latest ``mongodb/atlas`` Docker image.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         docker pull mongodb/atlas:latest
+
+   .. step:: Run the Docker image in bash mode.
+
+      To learn more, see :ref:`atlas-cli-docker`.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         docker run -p 27777:27017 --privileged -it mongodb/atlas bash
+
+   .. step:: Create a local |service| deployment.
+
+      The following command should return a connection string.
+
+      **Example:**
+
+      .. code-block::
+
+         atlas deployments setup --bindIpAll --username root --password root --type local --force
```

Comment:
Suggestion: add a line after the command with a link to the atlas-deployments-setup command so they can see all of the available options and what they do